controller/src/main/java/org/jboss/as/controller/operations/common/Util.java

```diff
@@ -179,16 +178,83 @@ public static ModelNode validateOperation(ModelNode operation) {
             try {
                 operation.get(OP_ADDR).asList();
             } catch (IllegalArgumentException ex) {
-                if(errors.length() > 0) {
-                    errors.append(System.lineSeparator());
+                try {
+                    ModelNode address = parseAddress(operation.get(OP_ADDR).asString());
```

Comment:
this looks mighty odd.
why not check the type of OP_ADDR attribute? something like if (operation.get(OP_ADDR).getType() == ModelType.STRING and if it is string try to parse it as such, doing this with try |catch is just wrong